src/internal/packager/sbom/catalog.go

```diff
@@ -106,22 +127,31 @@ func Catalog(ctx context.Context, componentSBOMs map[string]*layout.ComponentSBO
 	currComponent := 1
 
 	// Generate SBOM for each component
+	l.Info("creating SBOMs for components", "components", componentCount)
 	for component := range componentSBOMs {
+		// TODO(mkcp): Remove message on logger release
 		builder.spinner.Updatef("Creating component file SBOMs (%d of %d): %s", currComponent, componentCount, component)
+		l.Info("creating component file SBOMs", "component", component)
 
 		if componentSBOMs[component] == nil {
+			// TODO(mkcp): Remove message on logger release
 			message.Debugf("Component %s has invalid SBOM, skipping", component)
+			l.Debug("component has invalid SBOM, skipping", "component", component)
 			continue
 		}
 
 		jsonData, err := builder.createFileSBOM(ctx, *componentSBOMs[component], component)
 		if err != nil {
+			// TODO(mkcp): Remove message on logger release
 			builder.spinner.Errorf(err, "Unable to create SBOM for component %s", component)
+			l.Error("unable to create SBOM for component", "component", component, "error", err.Error())
```

Comment:
While the current code does print the error on the spinner, IMO it makes more sense to craft and return an error here, otherwise we will end up printing it out twice. I'm guessing it was sent to `spinner.Errorf` so the spinner would have the icon showing it failed.

Reply:
same comment for the other `builder.spinner.Errorf` calls

Reply:
fixed in latest